video/out/vo_drm.c

```diff
@@ -320,21 +314,33 @@ static void draw_image(struct vo *vo, mp_image_t *mpi)
             src_rc.x0 = MP_ALIGN_DOWN(src_rc.x0, mpi->fmt.align_x);
             src_rc.y0 = MP_ALIGN_DOWN(src_rc.y0, mpi->fmt.align_y);
             mp_image_crop_rc(&src, src_rc);
+
+            w = p->dst.x1 - p->dst.x0;
+            h = p->dst.y1 - p->dst.y0;
+            x = (p->cur_frame->w - w) >> 1;
```

Comment:
I tried to follow the same coding style of the original file, which had these variables signed and >>1 instead of /2. I agree that at least w and h should be unsigned, and dividing by two is clearer than shifting.